packages/safe-apps-web3modal/package.json

```diff
@@ -29,7 +29,6 @@
     "web3modal": "^1.9.3"
   },
   "peerDependencies": {
-    "@gnosis.pm/safe-apps-sdk": "4.0.0",
```

Comment:
Shouldn't this be a dependency instead of a devDependency?
It is imported here: https://github.com/gnosis/safe-apps-sdk/blob/master/packages/safe-apps-web3modal/src/modal.ts#L1

Reply:
there's currently a mess going on with dependencies 🤦 , it's already a dependency of a dependency package

Reply:
I'll just remove this change and keep it like it was because it worked 😄